Topics/User_Experience/Mobile-First_Design.md

```diff
@@ -0,0 +1,53 @@
+# Mobile-First Design for Responsive Web Apps
+
+## What is it?
+
+Mobile-first design is an approach that prioritizes designing the UI of web apps for mobile devices first before other larger screens. 
+
+The reasoning behind this strategy is to promote _progressive enhancement_ — the UI’s mobile version will contain only the main functionalities required for the web app, and adding extra features that can be displayed on larger screens can come later. In contrast, using the method of desktop-first design and adding responsiveness after is called _graceful degradation _because a lot of the details and complexities incorporated in the desktop version of the UI will have to be simplified and condensed to fit on a much smaller screen.
+
+## Benefits
+
+
+
+* Enhanced UX for mobile users
+    * As accessing web apps through phones has become the dominant mode of interaction — with more than [50%](https://www.statista.com/statistics/277125/share-of-website-traffic-coming-from-mobile-devices/#:~:text=Mobile%20accounts%20for%20approximately%20half,since%20the%20beginning%20of%202017.) of web traffic coming from mobile devices since 2022 — creating UIs with the mobile-first design approach ensures that the UX for mobile devices is not compromised from graceful degradation
+* Prioritized content
+    * With a smaller screen, the UI for mobile devices will likely have to cut out any unnecessary details or rearrange content, allowing your designs to be cleaner and more concise
+    * For instance, having a large block of text that spans your entire phone screen might lead you to shorten the text or break it up for an improved UX; the lengthiness of that body of text might have been overlooked initially if using the desktop-first approach
+* Improved accessibility
+    * Mobile-first design emphasizes legible typography and text sizes, ensuring readability and navigability for your web app
+* Reduced redesigning
+    * Adopting this approach can reduce the need for extensive redesign efforts later in the process if some critical features on the desktop version, such as hover effects, cannot be easily translated into the mobile version
+* Faster page loading times
+    * With a reduced amount of code, images, and resources required for the web app, losing speed can also be reduced
+
+## Drawbacks
+
+* Learning curve
+    * Most web app developers are accustomed to the desktop-first approach, thus shifting to this new strategy that requires a different way of thinking and planning requires some effort
+* Restricted design creativity and flexibility
+    * Mobile designs are more limited than desktop versions, which may put more restrictions on the content and features you can include
```

Comment:
Perhaps you can elaborate on this drawback explaning how "mobile designs are more limited than desktop versions". For example, you can include how complex data visualizations or intricate user interactions that are feasible on desktop versions are challenging on mobile versions.